Promact.Oauth.Server/src/Promact.Oauth.Server/Controllers/ConsumerAppController.cs

```diff
@@ -148,33 +150,28 @@ public ConsumerAppController(IConsumerAppRepository iConsumerAppRepository, User
         * HTTP/1.1 200 OK 
         * {
         *
-        *   "description":"retun true if succesfully consumer updated If any problem in updte consumer mehtod or not update consumer apps so return false."
+        *   "description":"return true if succesfully consumer app has been updated else return false."
         * }
         */
-        [HttpPost]
-        [Route("updateConsumer")]
-        public async Task<IActionResult> UpdateConsumerApps([FromBody]ConsumerAppsAc consumerAppsAc)
+        [HttpPut]
+        [Route("")]
+        public async Task<IActionResult> UpdateConsumerApp([FromBody]ConsumerAppsAc consumerAppsAc)
         {
             try
             {
-                var oldAppsDetails = await _consumerAppRepository.GetConsumerAppsById(consumerAppsAc.Id);
-                if (oldAppsDetails != null)
-                {
-                    oldAppsDetails.Name = consumerAppsAc.Name;
-                    oldAppsDetails.CallbackUrl = consumerAppsAc.CallbackUrl;
-                    oldAppsDetails.Description = consumerAppsAc.Description;
-                    oldAppsDetails.UpdatedDateTime = DateTime.Now;
-                    if (await _consumerAppRepository.UpdateConsumerApps(oldAppsDetails) != 0)
-                        return Ok(true);
-                    else
-                        return Ok(false);
-                }
-                return Ok(oldAppsDetails);
+                ConsumerApps consumerApp = await _consumerAppRepository.GetConsumerAppById(consumerAppsAc.Id);
+                consumerApp.Name = consumerAppsAc.Name;
+                consumerApp.CallbackUrl = consumerAppsAc.CallbackUrl;
+                consumerApp.Description = consumerAppsAc.Description;
+                consumerApp.UpdatedDateTime = DateTime.Now;
+                if (await _consumerAppRepository.UpdateConsumerApps(consumerApp) != 0)
+                    return Ok(true);
+                else
+                    return Ok(false);
             }
-            catch (Exception ex)
+            catch (Exception)
```

Comment:
I think sometimes unexpected database error may occur, at that time generic exception is thrown.